### PR TITLE
Update documentation screenshots

### PR DESCRIPTION
**Pull request by zepfietje**
## Description

When generating the new screenshots for https://github.com/filamentphp/filament/pull/11662, I noticed the wrong font weight was used (custom weights in Tailwind config from when we tweaked them for the old font). I've fixed the config in the section footer actions PR, since new screenshots needed to be added there. Then I regenerated all documentation screenshots and split it out into this PR. So this PR still depends on https://github.com/filamentphp/filament/pull/11662 to be merged first.

- [x] Visual changes (if any) are shown using screenshots/recordings of before and after.

## Code style

- [x] `composer cs` command has been run.

## Testing

- [x] Changes have been tested.

## Documentation

- [x] Documentation is up-to-date.

### CODE REVIEW

File: docs-assets/screenshots/images/light/tables/reordering/custom-trigger-action.jpg
Comment (zepfietje) [inline docs-assets/screenshots/images/light/tables/reordering/custom-trigger-action.jpg:1]: @danharrin, we probably don't want to display this one entirely in the docs. 😂

File: docs-assets/screenshots/images/light/tables/reordering.jpg
Comment (zepfietje) [inline docs-assets/screenshots/images/light/tables/reordering.jpg:1]: https://github.com/filamentphp/filament/pull/11670/files#r1510768918

File: docs-assets/screenshots/images/dark/tables/reordering.jpg
Comment (zepfietje) [inline docs-assets/screenshots/images/dark/tables/reordering.jpg:1]: https://github.com/filamentphp/filament/pull/11670/files#r1510768918

**Comment by zepfietje** [comment]
@danharrin I think you forgot to build the assets before regenerating the reordering screenshots? Font weight is wrong on the latest ones. 😄

**Comment by danharrin** [comment]
Redone them

**Comment by zepfietje** [comment]
Thanks!

**Pull request merged**